radio/src/lua/api_general.cpp

```diff
@@ -553,6 +553,26 @@ static int luaGetRotEncSpeed(lua_State * L)
   return 1;
 }
 
+/*luadoc
+@function getRotEncInvert()
+
+Return rotary encoder inverted status
+
+@retval number in list: OFF = 0, ON = 1, V-N = 2, V-A = 3
+  return 0 on radio without rotary encoder
+
+@status current Introduced in 2.20
```

Comment:
I think this should be 2.8.0? ;)

Reply:
Correct, should be 2.8.0. Not sure where that came from, mixed up with Companion version 220 perhaps.